haystack/nodes/file_classifier/file_type.py

```diff
@@ -33,12 +33,9 @@ def __init__(self, supported_types: List[str] = DEFAULT_TYPES):
         """
         Node that sends out files on a different output edge depending on their extension.
 
-        :param supported_types: the file types that this node can distinguish.
-            Note that it's limited to a maximum of 10 outgoing edges, which
-            correspond each to a file extension. Such extension are, by default
-            `txt`, `pdf`, `md`, `docx`, `html`. Lists containing more than 10
-            elements will not be allowed. Lists with duplicate elements will
-            also be rejected.
+        :param supported_types: The file types that this node can distinguish between.
+             The extension are, by default, `txt`, `pdf`, `md`, `docx`, and `html`.
```

Comment:
```suggestion
             The default values are: `txt`, `pdf`, `md`, `docx`, and `html`.
```

Reply:
Maybe worth mentioning that duplicates are not allowed? Or is that obvious?